packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/amplify-dynamo-model-resource-generator.ts

```diff
@@ -63,15 +62,15 @@ export class AmplifyDynamoModelResourceGenerator extends DynamoModelResourceGene
 
     // lambda that will handle DDB CFN events
     const gsiOnEventHandler = new aws_lambda.Function(scope, ResourceConstants.RESOURCES.TableManagerOnEventHandlerLogicalID, {
-      runtime: aws_lambda.Runtime.NODEJS_18_X,
+      runtime: aws_lambda.Runtime.NODEJS_16_X,
       code: lambdaCode,
       handler: 'amplify-table-manager-handler.onEvent',
       timeout: Duration.minutes(14),
     });
 
     // lambda that will poll for provisioning to complete
     const gsiIsCompleteHandler = new aws_lambda.Function(scope, ResourceConstants.RESOURCES.TableManagerIsCompleteHandlerLogicalID, {
-      runtime: aws_lambda.Runtime.NODEJS_18_X,
+      runtime: aws_lambda.Runtime.NODEJS_16_X,
```

Comment:
Why did we downgrade node version?

Reply:
The Node 18 in lambda does not support aws sdk v2. This will be bumped once the sdk is bumped to use v3.